spec/es_spec_helper.rb

```diff
@@ -0,0 +1,42 @@
+require "logstash/devutils/rspec/spec_helper"
+require "ftw"
+require "logstash/plugin"
+require "logstash/json"
+require "stud/try"
+require "longshoreman"
+
+module ESHelper
+  def get_host
+    Longshoreman.new.get_host_ip
+  end
+
+  def get_port(protocol)
+    container = Longshoreman::Container.new
+    container.get("elasticsearch")
+    case protocol
+    when "http"
+      container.rport(9200)
+    when "transport", "node"
+      container.rport(9300)
+    end
+  end
+
+  def get_client
+    Elasticsearch::Client.new(:host => "#{get_host}:#{get_port('http')}")
+  end
+end
+
+RSpec.configure do |config|
+  config.include ESHelper
+
+  config.before(:suite, :integration => true) do
+    @docker = Longshoreman.new("elasticsearch:1.6", "elasticsearch")
```

Comment:
Having the image and tag named in a constant is my preferred way to do it:

```
    @docker = Longshoreman.new("#{IMAGE}:#{TAG}", NAME)
```

It makes it less likely you'll forget to bump a version, having it defined as a constant near the top.